### PR TITLE
chore: check for README.md file inside the extension

### PR DESCRIPTION
**Pull request by benoitf**
related to https://github.com/containers/podman-desktop/issues/7363

Change-Id: I0e5eb4a2736fcd89906057f445302f27fcdbab02

### CODE REVIEW

File: .github/workflows/pr-check.yaml [hunk 1]
```diff
@@ -35,7 +35,18 @@ jobs:
                   for i in `jq -r .extensions[].versions[].ociUri api/extensions.json`
                   do
                     echo "Verify image $i"
-                    podman pull -q $i
+                    CONTAINER_ID=$(podman create -q --entrypoint sh $i)
```
Comment (jeffmaury) [inline .github/workflows/pr-check.yaml:38]: Wondering if this would work as extension images are created from scratch ?

Comment (benoitf) [reply .github/workflows/pr-check.yaml:38]: yes it works

Comment (benoitf) [reply .github/workflows/pr-check.yaml:38]: in fact on recent podman version you don't need to provide the entrypoint on create but on Ubuntu's version if you don't you have an error saying that the entrypoint is missing ...

Comment (benoitf) [reply .github/workflows/pr-check.yaml:38]: note: it's a create command, not a run/start command

**Review by jeffmaury — APPROVED** [review]
LGTM

**Review by deboer-tim — APPROVED** [review]

**Pull request merged**